process-controller/src/main/java/org/jboss/as/process/logging/ProcessLogger.java

```diff
@@ -433,23 +433,23 @@
      *
      * @return the message.
      */
-    @Message(id = Message.NONE, value = "Set the host controller's running type to ADMIN_ONLY causing it to open administrative interfaces and accept management requests but not start servers or, if this host controller is the master for the domain, accept incoming connections from slave host controllers.")
+    @Message(id = Message.NONE, value = "Set the host controller's running type to ADMIN_ONLY causing it to open administrative interfaces and accept management requests but not start servers or, if this host controller is the primary for the domain, accept incoming connections from secondary host controllers.")
     String argAdminOnly();
 
     /**
      * Instructions for the {@link CommandLineConstants#MASTER_ADDRESS} command line argument.
      *
      * @return the message.
      */
-    @Message(id = Message.NONE, value = "Set system property jboss.domain.master.address to the given value. In a default slave Host Controller config, this is used to configure the address of the master Host Controller.")
+    @Message(id = Message.NONE, value = "Set system property jboss.domain.primary.address to the given value. In a default secondary Host Controller config, this is used to configure the address of the primary Host Controller.")
```

Comment:
Don't change the property name, here we are changing only messages of the loggers, properties are tracked on a different Jira.